views/js/back.js

```diff
@@ -246,7 +268,7 @@ $(window).ready(function () {
     });
 
     // Tab Content : Edit : Redirect
-    $(document).on('change', 'input[name="PSR_REDIRECTION_1"],input[name="PSR_REDIRECTION_2"],input[name="PSR_REDIRECTION_3"]', (e) => {
+    $(document).on('change', 'input[name^="PSR_REDIRECTION_"]', (e) => {
```

Comment:
👍

Reply:
I recently learnt this notation but this is really handy!

Reply:
:+1:

Reply:
:+1: